zebrad/tests/acceptance.rs

```diff
@@ -83,11 +83,17 @@ fn default_test_config() -> Result<ZebradConfig> {
         ..mempool::Config::default()
     };
 
+    let consensus = zebra_consensus::Config {
+        debug_skip_parameter_preload: true,
+        ..zebra_consensus::Config::default()
+    };
```

Comment:
Code Coverage: This is a good default for most tests, but `sync_past_mandatory_checkpoint` should enable the pre-download, because it actually does Sprout and Sapling verification.

Reply:
https://github.com/ZcashFoundation/zebra/pull/3197/commits/032d95543ceae392e2c3fdc0de6e105b413c70b9

Reply:
`create_cached_database_height` is run by both `create_cached_database` and `sync_past_mandatory_checkpoint`.

`create_cached_database` doesn't use or cache the parameters, and we want the sync to be fast, so we don't want the parameter download task running.

`sync_past_mandatory_checkpoint` does use the parameters, so the pre-download is useful.